sounds/management/commands/list_sound_analysis.py

```diff
@@ -0,0 +1,41 @@
+#
+# Freesound is (c) MUSIC TECHNOLOGY GROUP, UNIVERSITAT POMPEU FABRA
+#
+# Freesound is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Freesound is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#     See AUTHORS file.
+#
+
+import json
+import logging
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from sounds.models import Sound, SoundAnalysis
+
+console_logger = logging.getLogger("console")
+
+
+class Command(BaseCommand):
+
+    help = 'Prints on screen a list of all SoundAnalysis objects and some of their details (the sound they refer to, the extractor name and version)'
+
+    def handle(self, *args, **options):
+        console_logger.info("Obtaining SoundAnalysis list...")
+        for a in SoundAnalysis.objects.all():
+            print(json.dumps({'SoundAnalysis_id': a.id, 'sound_id': a.sound.id, 
```

Comment:
Why are you using `json.dumps` here instead of simply printing or using string formatting?

Reply:
There is no proper reason for it... I was just using `json.dumps` a lot (in "models.analyze_new" method, in "gm_worker_analysis.py"), so I did it here as well because it gets printed all right. It could be a simple string, indeed :)

Reply:
IIRC, the reason `json.dumps` is used in some parts where the workers communicate with the Django code is because that is the way we serialize parameters and pass to the workers.

Reply:
Yes, you recall correctly, but also in some loggers info (for instance `workers_logger.info` in "gm_worker_processing"). So when I saw it was printed nicely, I just kept using it – it does not make a lot of sense, as you point out.